programs/ssl/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@ set(executables
     ssl_server
     ssl_server2
 )
+add_dependencies(${programs_target} ${executables})
+add_dependencies(${ssl_opt_target} ${executables})
```

Comment:
Minor: I think this could be written in one line?

Reply:
How? This is appending the same list to two different variables.

Reply:
That's right, sorry. Got a bit confused in the end.